### PR TITLE
Create Ofo.so.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Ofo.so.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!--
+	Non-functional hosts
+		Couldn't connect to server:
+			 - cms.ofo.so
+			 - mail.ofo.so
+
+		4xx client error:
+			 - common.ofo.so
+			 - img.ofo.so
+			 - one.ofo.so
```
Comment (J0WI) [inline src/chrome/content/rules/Ofo.so.xml]: Try custom test urls, like `https://common.ofo.so/app/`

Comment (cschanaj) [inline src/chrome/content/rules/Ofo.so.xml]: done. thanks!

Comment (J0WI) [inline src/chrome/content/rules/Ofo.so.xml]: `http://img.ofo.so/apk/`
You can easily search for test urls using search engines.

Comment (cschanaj) [inline src/chrome/content/rules/Ofo.so.xml]: Sorry that I cannot find working URL(s) for `img` and `one` other than the one you have suggested. I tried Google `inurl:img.ofo.so`, `site:img.ofo.so` and even `img.ofo.so`, none of these give me a working URL. What kind of search query  did you use?

Comment (J0WI) [inline src/chrome/content/rules/Ofo.so.xml]: I used "img.ofo.so" :P

**Pull request merged**